GitUI/UserEnvironmentInformation.cs

```diff
@@ -35,12 +35,13 @@ public static string GetInformation()
             // Build and open FormAbout design to make sure info still looks good if you change this code.
             StringBuilder sb = new StringBuilder();
 
-            sb.Append($"- Git Extensions {AppSettings.ProductVersion}{Environment.NewLine}");
-            sb.Append($"- {_sha} {(_dirty ? " (Dirty)" : "")}{Environment.NewLine}");
-            sb.Append($"- Git {gitVer}{Environment.NewLine}");
-            sb.Append($"- {Environment.OSVersion}{Environment.NewLine}");
-            sb.Append($"- {RuntimeInformation.FrameworkDescription}{Environment.NewLine}");
-            sb.Append($"- DPI X:{DpiUtil.ScaleX:P} Y:{DpiUtil.ScaleY:P}");
+            sb.AppendLine($"- Git Extensions {AppSettings.ProductVersion}");
+            sb.AppendLine($"- Build {_sha.Substring(0, 9)}{(_dirty ? " (Dirty)" : "")}");
```

Comment:
The risk for ambiguity is low, but I still want o have the full sha or git to shorten the ref

Reply:
I don't want even a git shortened sha here.  I want the complete sha always.  I chose to do it that way on purpose.  You don't know if they are documenting something that needs to stand the passage of time.  Do not shorten at all this sha.  Github will shorten on display which is fine.  

![image](https://user-images.githubusercontent.com/3629489/48446033-2a782f00-e766-11e8-8dde-2295dec2f6a1.png)